sdk/go/common/workspace/plugins.go

```diff
@@ -388,9 +388,7 @@ func newGitHTTPSSource(url *url.URL) (*gitSource, error) {
 func (source *gitSource) GetLatestVersion(
 	ctx context.Context, _ func(*http.Request) (io.ReadCloser, int64, error),
 ) (*semver.Version, error) {
-	// TODO: support this for the users convenience
-	return nil,
-		errors.New("GetLatestVersion is not supported for git sources, please specify the version of the plugin")
+	return gitutil.GetLatestTagOrHash(ctx, source.url)
```

Comment:
How hard is it to add an integration test?

Reply:
The best way would probably be to not have a full integration test, but just create a fake `gitSource` with a local repository. I think I can do that.